### PR TITLE
DOC-191: Recording JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namespaces:
- Recording

Other:
- Fix type in Assets JSDoc.

Case: https://highfidelity.atlassian.net/browse/DOC-191

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21873/HighFidelity-Beta-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21873/HighFidelity-Beta-Interface-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21873/HighFidelity-Beta-PR16335-1715a86.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21873/HighFidelity-Quest-Beta-PR16335-1715a86.apk)

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21879/HighFidelity-Beta-PR16335-1715a86.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21879/HighFidelity-Quest-Beta-PR16335-1715a86.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21879/HighFidelity-Beta-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21879/HighFidelity-Beta-Interface-PR16335-1715a86.dmg

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21911/HighFidelity-Beta-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21911/HighFidelity-Beta-Interface-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21911/HighFidelity-Beta-PR16335-1715a86.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21911/HighFidelity-Quest-Beta-PR16335-1715a86.apk)

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21920/HighFidelity-Beta-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21920/HighFidelity-Beta-Interface-PR16335-1715a86.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21920/HighFidelity-Beta-PR16335-1715a86.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21920/HighFidelity-Beta-Interface-PR16335-1715a86.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21920/HighFidelity-Beta-PR16335-1715a86.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21920/HighFidelity-Quest-Beta-PR16335-1715a86.apk)

File: libraries/script-engine/src/RecordingScriptingInterface.h [hunk 1]
```diff
@@ -102,132 +128,222 @@ public slots:
...
      */
     float recorderElapsed() const;
 
 
     /**jsdoc
+     * Gets the default directory that recordings are saved in.
      * @function Recording.getDefaultRecordingSaveDirectory
-     * @returns {string}
+     * @returns {string} The default recording save directory.
+     * @exampl <caption>Report the default save directory.</caption>
```
Comment (jennaingersoll) [inline libraries/script-engine/src/RecordingScriptingInterface.h]:
```suggestion
     * @example <caption>Report the default save directory.</caption>
```

Comment (ctrlaltdavid) [reply libraries/script-engine/src/RecordingScriptingInterface.h]: Fixed.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22078/HighFidelity-Beta-PR16335-777f969.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22078/HighFidelity-Beta-Interface-PR16335-777f969.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22078/HighFidelity-Beta-PR16335-777f969.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22078/HighFidelity-Quest-Beta-PR16335-777f969.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22078/HighFidelity-Beta-PR16335-777f969.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22078/HighFidelity-Beta-Interface-PR16335-777f969.exe

**Pull request merged**